PiedPiper.xcodeproj/GeneratedModuleMap/NimbleCwlCatchExceptionSupport/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module NimbleCwlCatchExceptionSupport {
```

Comment:
Why are these modulemaps added?

Reply:
these files are generated by running `swift package generate-xcodeproj`. This is required for the package to work properly with Carthage.

Reply:
@toupper I've removed the auto generated xcodeproj file and manually created the new one which doesn't have any auto-generated files